src/framework/standard/mod.rs

```diff
@@ -53,12 +53,12 @@ use client::CACHE;
 #[cfg(feature = "cache")]
 use model::channel::Channel;
 
-/// A convenience macro for generating a struct fulfilling the [`Command`] trait.
+/// A convenience macro for generating a struct fulfilling the [`Command`](framework/standard/trait.Command.html) trait.
```

Comment:
same

Reply:
It does not work here because the compiler treats these names case-__in__sensitive so it would link to command method.